aries-site/src/pages/design-tokens/all-design-tokens.js

```diff
@@ -64,10 +67,48 @@ const Nav = ({ active, setActive, tokens: tokensObj }) => {
   ));
 };
 
+const NavPane = ({ children }) => {
+  const theme = useContext(ThemeContext);
+  return (
+    <Box
+      flex={false}
+      style={{
+        position: 'sticky',
+        top: theme.global.edgeSize.medium,
+      }}
+      height="130vh"
```

Comment:
I see why you added this `height` but also see an issue this is causing 


https://github.com/user-attachments/assets/c927ab17-83f5-413f-bf9f-3dad70d944a8

Reply:
bummer. I thought I had addressed that. Lost it along the way. Will fix.

Reply:
fixed.